README.md

```diff
@@ -133,12 +133,122 @@ If you have all of your components in the folder `./comps` then you would call `
 
 Every folder directly under `./comps` would get processed and the build tools and handed off to rollup.
```

Comment:
There is something a bit off about this part of the sentence, but I'm not sure how to fix it: `and the build tools and handed off to rollup`

Reply:
How about this:
> Every folder directly under `./comps` would get processed. Any subfolder that was found to match the requirements of a component will be handed off to rollup for further processing.